Makefile

```diff
@@ -103,3 +103,16 @@ binaries-all-sys:
 	GOOS=linux   GOARCH=amd64 $(MAKE) binaries
 	GOOS=linux   GOARCH=arm64 $(MAKE) binaries
 	GOOS=windows GOARCH=amd64 $(MAKE) binaries
+
+.PHONY: update-dep
+update-dep:
+	for mod in $$(grep -l '$(MODULE) ' $$(find . -name go.mod)); do \
+		dir=$$(dirname $$mod); \
+		(cd $$dir && go get $(MODULE)@$(VERSION)); \
+	done
+	$(MAKE) otelcontribcol
+	$(MAKE) gotidy
+
+.PHONY: update-otel
+update-otel:
+	$(MAKE) update-dep MODULE=github.com/open-telemetry/opentelemetry-collector VERSION=master
```

Comment:
`$(MAKE) for-all  CMD="go get -u github.com/open-telemetry/opentelemetry-collector@0e1b2e3"`

You can also run this from external
`CMD="go get -u github.com/open-telemetry/opentelemetry-collector@0e1b2e3" make for-all`